rsocket-transport-netty/src/main/java/io/rsocket/transport/netty/WebsocketDuplexConnection.java

```diff
@@ -43,12 +47,20 @@ public WebsocketDuplexConnection(NettyInbound in, NettyOutbound out, NettyContex
 
   @Override
   public Mono<Void> sendOne(Frame frame) {
-    return out.sendObject(new BinaryWebSocketFrame(frame.content())).then();
+    ByteBuf content = frame.content().skipBytes(3);
+    return out.sendObject(new BinaryWebSocketFrame(content)).then();
   }
 
   @Override
   public Flux<Frame> receive() {
-    return in.receive().map(buf -> Frame.from(buf.retain()));
+    return in.receive().map(buf -> {
+      buf.retain();
+      CompositeByteBuf composite = context.channel().alloc().compositeBuffer();
+      ByteBuf length = Unpooled.wrappedBuffer(new byte[3]);
+      FrameHeaderFlyweight.encodeLength(length, 0, buf.readableBytes());
+      composite.addComponents(true, length, buf);
```

Comment:
If we would like to keep the 'buf' alive, it seems better to call 'retain' prior to giving it to 'addComponents' function. 

https://netty.io/4.0/api/io/netty/buffer/CompositeByteBuf.html
"buffer - the ByteBuf to add. ReferenceCounted.release() ownership is transfered to this CompositeByteBuf."